kalite/contentload/management/commands/unpack_assessment_zip.py

```diff
@@ -20,17 +26,30 @@
 )
 
 ASSESSMENT_ITEMS_PATH = os.path.join(KHAN_DATA_PATH, "assessmentitems.json")
+ASSESSMENT_ITEMS_VERSION_PATH = os.path.join(KHAN_DATA_PATH, "assessmentitems.json.version")
 
 logging = settings.LOG
 
 class Command(BaseCommand):
 
+    option_list = BaseCommand.option_list + (
+        make_option("-f", "--force",
```

Comment:
If this flag isn't specified, we will always check if we need to download the latest assessment zip file.